.chloggen/remove-signalfx-forwarder.yaml

```diff
@@ -0,0 +1,8 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: agent
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove smartagent/signalfx-forwarder from the agent default config.
```

Comment:
Can we add the guidance in PR description to the subtext of changelog so the info is more visible for users?

Reply:
I was thinking it isn't worth bloating the changelog given that it already has a link to PR, but I can add it if you think otherwise.

Reply:
Ah, that works too. I always want more info everywhere 😅